conf/application.conf

```diff
@@ -81,7 +81,3 @@ goods-item-records {
 trader-profiles {
   ttl = 365 days
 }
-
-features {
-     patchMethodEnabled = false
```

Comment:
Is this aligned with  @william-eyre updates?

Reply:
This pr is to remove the changes done previously to add feature flags to the hawk stub.